### ISSUE TITLE
Error in connecting with Magento. Error Message: The server sent HTTP status code 200: OK

### ISSUE THREAD

**Issue by jesse-thomas** [hotwaxlabs/magento#2]
I followed the steps in Magento-Setup-Guide.pdf and am getting the error below: 

"Error in connecting with Magento. Error Message: The server sent HTTP status code 200: OK"

I'm using OFBiz 13.07.02 and Magento 1.9.1.0

I started to debug but can't find the source for the class MageApiModelServerWsiHandlerPortType.class from MagentoSOAP.jar

**Comment by arunpati** [comment]
Thanks Jesse for reporting the issue.  Please use magneto version 1.9.0. We are working to provide support for latest version of Magento.

Please let us know if you face issue with Magento 1.9.0 as well.

**Comment by jesse-thomas** [comment]
I setup Magento 1.9.0.0 with magneto sample data 1.9.0.0 and no luck. I still get same error "..status code 200..". should I try with 1.8?

**Comment by arunpati** [comment]
Hi Jesse,

Please share more details regarding your issue. Like 
-- The URL pattern that you are using to connect with Magento. 
-- Steps to regenerate ths issue. 
-- Console logs if any.

**Comment by jesse-thomas** [comment]
http://<magnetohost>/api/v2_soap?wsdl
AND
http://<magnetohost>/index.php/api/v2_soap?wsdl

I log into OFBiz then go to https://localhost:8443/magento/control/IntegrationManager

enter the Magento Connect Url, SOAP User Name, Password 

Console output:

2015-11-30 19:13:44,917 |-0.0.0.0-8443-exec-9 |ServiceDispatcher             |T| Sync service [magento/createMagentoConfiguration] finished in [50] milliseconds
2015-11-30 19:13:53,599 |-0.0.0.0-8443-exec-9 |MagentoServices               |I| The server sent HTTP status code 200: OK
2015-11-30 19:13:53,600 |-0.0.0.0-8443-exec-9 |ServiceDispatcher             |E| Error in Service [testMagentoConnection]: Error in connecting with Magento. Error Message: The server sent HTTP status code 200: OK
2015-11-30 19:13:53,600 |-0.0.0.0-8443-exec-9 |TransactionUtil               |W| [TransactionUtil.setRollbackOnly] Calling transaction setRollbackOnly; this stack trace shows where this is happening:
java.lang.Exception: Error in Service [testMagentoConnection]: Error in connecting with Magento. Error Message: The server sent HTTP status code 200: OK
	at org.ofbiz.entity.transaction.TransactionUtil.setRollbackOnly(TransactionUtil.java:378) [ofbiz-entity-test.jar:?]
	at org.ofbiz.entity.transaction.TransactionUtil.rollback(TransactionUtil.java:319) [ofbiz-entity-test.jar:?]
	at org.ofbiz.service.ServiceDispatcher.runSync(ServiceDispatcher.java:539) [ofbiz-service.jar:?]
	at org.ofbiz.service.ServiceDispatcher.runSync(ServiceDispatcher.java:232) [ofbiz-service.jar:?]
	at org.ofbiz.service.GenericDispatcherFactory$GenericDispatcher.runSync(GenericDispatcherFactory.java:83) [ofbiz-service.jar:?]
	at org.ofbiz.magento.MagentoServices.createUpdateMagentoConfiguration(MagentoServices.java:400) [Magento.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.7.0_71]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[?:1.7.0_71]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.7.0_71]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[?:1.7.0_71]
	at org.ofbiz.service.engine.StandardJavaEngine.serviceInvoker(StandardJavaEngine.java:100) [ofbiz-service.jar:?]
	at org.ofbiz.service.engine.StandardJavaEngine.runSync(StandardJavaEngine.java:57) [ofbiz-service.jar:?]
	at org.ofbiz.service.ServiceDispatcher.runSync(ServiceDispatcher.java:400) [ofbiz-service.jar:?]
	at org.ofbiz.service.ServiceDispatcher.runSync(ServiceDispatcher.java:232) [ofbiz-service.jar:?]
	at org.ofbiz.service.GenericDispatcherFactory$GenericDispatcher.runSync(GenericDispatcherFactory.java:83) [ofbiz-service.jar:?]
	at org.ofbiz.webapp.event.ServiceEventHandler.invoke(ServiceEventHandler.java:341) [ofbiz-webapp.jar:?]
	at org.ofbiz.webapp.control.RequestHandler.runEvent(RequestHandler.java:696) [ofbiz-webapp.jar:?]
	at org.ofbiz.webapp.control.RequestHandler.doRequest(RequestHandler.java:421) [ofbiz-webapp.jar:?]
	at org.ofbiz.webapp.control.ControlServlet.doGet(ControlServlet.java:211) [ofbiz-webapp.jar:?]
	at org.ofbiz.webapp.control.ControlServlet.doPost(ControlServlet.java:90) [ofbiz-webapp.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:646) [servlet-api-3.0.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:727) [servlet-api-3.0.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.ofbiz.webapp.control.ContextFilter.doFilter(ContextFilter.java:317) [ofbiz-webapp.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:504) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:950) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:421) [tomcat-7.0.59-catalina.jar:7.0.59]
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1074) [tomcat-7.0.59-tomcat-coyote.jar:7.0.59]
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:611) [tomcat-7.0.59-tomcat-coyote.jar:7.0.59]
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:316) [tomcat-7.0.59-tomcat-coyote.jar:7.0.59]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145) [?:1.7.0_71]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615) [?:1.7.0_71]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-7.0.59-tomcat-coyote.jar:7.0.59]
	at java.lang.Thread.run(Thread.java:745) [?:1.7.0_71]

**Comment by jesse-thomas** [comment]
I was just messing around with postman, making requests to the Magento install. I looks like all I'm getting back is the wsdl file when I should be getting a response to the login request.

removing the ?wsdl from the Magento url gets me past the error in ofbiz and an expected response postman.

**Comment by jesse-thomas** [comment]
After more testing I think the cause of the issue was that **I did not have the php-soap extension installed**. After installing the soap extension for php Magento 1.9.0.0 worked for the login, but Magento was still trowing some other xml related errors when trying to sync (seemed sort of random). I was able to connect and sync categories and products.

I tried Magento 1.9.1.0 just for the hell of it and it worked first try with and without the ?wsdl on the url. If I come across anything else that may be usefull I'll add it here.
